lib/cdo/aws/cloudfront.rb

```diff
@@ -177,7 +177,8 @@ def self.distribution_config(app, origin, aliases, ssl_cert=nil)
               Id: app_name == proxy ? 'cdo' : app_name,
               CustomOriginConfig: {
                 OriginProtocolPolicy: 'match-viewer',
-                OriginSSLProtocols: %w(TLSv1.2 TLSv1.1)
+                OriginSSLProtocols: %w(TLSv1.2 TLSv1.1),
+                OriginReadTimeout: rack_env?(:levelbuilder) ? 60 : 30
```

Comment:
Minor suggestion: Maybe we should conditionally specify the `OriginReadTimeout` setting and let CloudFront use its default setting instead of explicitly specifying what is currently the default value.

Reply:
I am worried this will add a fair bit of complexity without gaining us much. If you would prefer to go that route, it's fine with me, but in that case I would love a more specific idea of how this might work.

when I try the specific suggestion above, my editor complains because I am adding erb syntax to a .rb file. if I try to rename it to .rb.erb, then I get this error:
```
Dave-MBP:~/src/cdo2 (levelbuilder-read-timeout)$ bundle exec rake adhoc:cdn:validate RAILS_ENV=adhoc VALIDATE=1
rake aborted!
LoadError: cannot load such file -- cdo/aws/cloudfront
/Users/dsb/src/cdo2/lib/cdo/cloud_formation/cdo_app.rb:11:in `require'
...
```
it's certainly possible that this could be made to work, but since I'm not clear what the ROI is here I stopped there.

Reply:
@sureshc since you gave approval and I don't see an obvious way to incorporate your feedback, I'm going to go ahead and merge what I have. please let me know if I can be of assistance in a follow up PR to avoid explicitly setting the default value of this param.

Reply:
Oops. I forgot this isn't an ERB. The change I was thinking of is not important :)
Thanks so much for taking this on!